test/Confluent.Kafka.IntegrationTests/AdminClient_DescribeTopics.cs

```diff
@@ -0,0 +1,189 @@
+                    }
+                );
+                var descResWithAuthOps = await adminClient.DescribeTopicsAsync(
+                    topicListAuthOps,
+                    describeOptionsWithAuthOps);
+                Assert.NotEmpty(descResWithAuthOps.TopicDescriptions[0].AuthorizedOperations);
+                Assert.Equal(8, descResWithAuthOps.TopicDescriptions[0].AuthorizedOperations.Count);
+
+                var topicACLs =  new List<AclBinding>
+                {
+                    new AclBinding()
+                    {
+                        Pattern = new ResourcePattern
+                        {
+                            Type = ResourceType.Topic,
+                            Name = singlePartitionTopic,
+                            ResourcePatternType = ResourcePatternType.Literal
+                        },
+                        Entry = new AccessControlEntry
+                        {
+                            Principal = "User:user",
+                            Host =  "*",
+                            Operation = AclOperation.Read,
+                            PermissionType = AclPermissionType.Allow
+                        }
+                    },
+                    new AclBinding()
+                    {
+                        Pattern = new ResourcePattern
+                        {
+                            Type = ResourceType.Topic,
+                            Name = singlePartitionTopic,
+                            ResourcePatternType = ResourcePatternType.Literal
+                        },
+                        Entry = new AccessControlEntry
+                        {
+                            Principal = "User:user",
+                            Host =  "*",
+                            Operation = AclOperation.Alter,
+                            PermissionType = AclPermissionType.Allow
+                        }
+                    },
+                };
+
+                var createAclsOptions = new CreateAclsOptions
+                {
+                    RequestTimeout = TimeSpan.FromSeconds(30)
+                };
+                await adminClient.CreateAclsAsync(topicACLs, createAclsOptions);
+
+                using (var adminClientUser = new AdminClientBuilder(new AdminClientConfig
+                {
+                    SecurityProtocol = SecurityProtocol.SaslPlaintext,
+                    SaslMechanism = SaslMechanism.Plain,
+                    SaslUsername = user,
+                    SaslPassword = userSecret,
+                    BootstrapServers = bootstrapServers
+                }).Build())
+                {
+                     descResWithAuthOps = await adminClientUser.DescribeTopicsAsync(topicListAuthOps,
+                        describeOptionsWithAuthOps);
+                }
+
+                var descResAuthOps =
+                    descResWithAuthOps.TopicDescriptions[0].AuthorizedOperations;
+                Assert.Equal(
```

Comment:
I think we should OrderBy before asserting equality, because while the order of the operations will be exactly what you described, there is no reason for it to be so (and it's not something we or the user should assume)
Here and elsewhere

Reply:
The order will be that one because it comes from the order in the enum that corresponds to the bitfield order too